### DISCUSSION TITLE
Remove or disable default container registries?

### DISCUSSION THREAD

**Discussion by fillg1** [podman-desktop/podman-desktop#10283] [Q&A]
Hi folks
We are using podman desktop on MacOS and we want to achieve that all container images are loaded from our own container registry eg. registry.mycompany.com/docker without prepending the repository url for all containers. The company conatiner registry is already configured as proxy/cache ..

So when someone does eg.
podman pull alpine:latest it should pull registry.mycompany.com/docker/alpine:latest instead. 
Is this possible ?

Its easy to add new registries by simply adding podman login, but it is possible to disable the standard registries like docker.io?

**Comment by benoitf** [comment]
hello, it looks like there are several questions

> podman pull alpine:latest it should pull registry.mycompany.com/docker/alpine:latest instead.

it's possible to configure the VM where podman is running `podman machine ssh` and then change the resolution

the file is `/etc/containers/registries.conf` (inside the podman machine VM)

look for `unqualified-search-registries`  (and maybe registry.mirror for your usecase)

https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md#description




> Its easy to add new registries by simply adding podman login, but it is possible to disable the standard registries like docker.io?

if you disable the extension (registries) you should not see them
